langsecure/shield.py

```diff
@@ -32,6 +33,8 @@ def __init__(self, **params):
         else:
             # Load the policies into the pydantic class.
             self._py_policystore = store.PyPolicyStore(self.policy_store)
+        if self.tracking_server:
```

Comment:
can we always use the tracer ? By default it can use file based logging which can be in current dir or in ~/.langsecure/traces.log

If user specifies the path, we could log into that file or if user inits and passes langfuse server then we can push to langfuse.